test/plugin.test.js

```diff
@@ -54,7 +57,6 @@ QUnit.module('videojs-contrib-eme', {
 
   afterEach() {
     window.navigator.requestMediaKeySystemAccess = this.origRequestMediaKeySystemAccess;
-    this.player.dispose();
```

Comment:
If only a specific test needs to handle the dispose, it might be better to use a different QUnit module or manually create a new player at the beginning of that test.

Reply:
The `dispose` after each test was already here, so I distributed them through the tests. I don't think it's actually necessary for any of the tests below though. Is that your sense too? If so, I'll remove it from all of the other tests and keep it in the only test actually uses `dispose`.